extensions/grpc/runtime/src/main/java/io/quarkus/grpc/runtime/reflection/GrpcServerIndex.java

```diff
@@ -33,9 +33,9 @@ public GrpcServerIndex(
         Set<String> files = new HashSet<>();
         Set<String> names = new HashSet<>();
         Set<FileDescriptor> services = new HashSet<>();
-        Map<String, FileDescriptor> descriptorsByName = new LinkedHashMap<>();
-        Map<String, FileDescriptor> descriptorsBySymbol = new LinkedHashMap<>();
-        Map<String, Map<Integer, FileDescriptor>> descriptorsByExtensionAndNumber = new LinkedHashMap<>();
+        Map<String, FileDescriptor> descriptorsByName = new HashMap<>();
+        Map<String, FileDescriptor> descriptorsBySymbol = new HashMap<>();
+        Map<String, Map<Integer, FileDescriptor>> descriptorsByExtensionAndNumber = new HashMap<>();
```

Comment:
Insertion order was important. I need to see if that still the case. @michalszynkiewicz

Reply:
I think these LinkedHashMap objects are not traversed in the program. As I have seen, only containsKey() and get() methods are invoked.